### DISCUSSION TITLE
[Error] FL client execution exception in hello-pt example under POC mode

### DISCUSSION THREAD

**Discussion by hitesh-ag1** [NVIDIA/NVFlare#1165] [Q&A] [answered]
Hi,

I am still a beginner to NVFlare and am facing some issues using the POC mode. It would be great if some debugging advice could be provided.

I am trying to run the hello_pt example in POC mode as described in the following link: https://nvflare.readthedocs.io/en/main/examples/hello_pt.html. However, I am getting the error "Module cifar10trainer.Cifar10Trainer" not found even though the files are present under the 'admin/transfer/hello-pt/app/custom' folder. The server, admin and clients have all started correctly and the job was also submitted successfully after which I recieve the aforementioned error. The full error is as follows:

`worker_process - ERROR - FL client execution exception: Config error in ['/tmp/nvflare/poc/site-2/startup/../c6bd5e35-fe9d-4d40-916c-5386d2dec461/app_site-2/config/config_fed_client.json']: Error processing ['/tmp/nvflare/poc/site-2/startup/../c6bd5e35-fe9d-4d40-916c-5386d2dec461/app_site-2/config/config_fed_client.json'] in JSON element {'path': 'cifar10trainer.Cifar10Trainer', 'args': {'lr': 0.01, 'epochs': 1}}: path: executors.#1.executor, exception: Module cifar10trainer.Cifar10Trainer does not exist`

Steps to reproduce the error:
1. Setup POC environment with 2 clients
2. Create a "transfer" folder under admin of POC folder
3. Copy hello-pt folder under examples of the main repo to admin/transfer of POC folder
4. Start POC
5. Submit hello-pt job

System details - Ubuntu 20.04 LTS, Python 3.8.10, nvflare v2.2.2

**Comment by yhwen** [comment, marked as answer]
Hi hitesh-ag1, could you please include your hello-pt job for investigation? There have been some changes to the hello-pt example. It could be your testing version is from dev branch, not compatible with v2.2.2

**Reply by hitesh-ag1** [reply]
Thanks @yhwen , that was indeed the problem. After I switched the hello-pt job example from 'dev' branch to '2.2' branch, the error resolved itself.
